clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookMessageHandler.ts

```diff
@@ -47,8 +47,8 @@ export async function resolveCopybookHandler(
   dialectType: string,
 ): Promise<string | undefined> {
   let result: string | undefined;
-  const e4eApi = await E4ECopybookService.getE4EAPI();
-  if (e4eApi && e4eApi.isEndevorElement(documentUri)) {
+
+  if (await E4ECopybookService.getE4EClient(documentUri)) {
```

Comment:
isn't one of the original cases broken by this? When E4E is enabled, available and fails, the processing will now fallback to zowe.